help/user-guide/viewing-project-in-single-page.md

```diff
@@ -0,0 +1,13 @@
+---
+ title: SEEK User Guide - Viewing Project in Single Page
+ layout: page
+---
+
+# Experiment View
+
```

Comment:
Again, add disclaimer that this refers to a feature that needs to be turned on in the admin panel.